regex.test.js

```diff
@@ -26,4 +26,27 @@ describe('regex', () => {
 		console.log({ match });
 		expect(!!match).toBe(true);
 	});
+	it('match hash/cash tags', () => {
+		// In order - No hashtag, Chinese, Hindi, Spanish, French, Russian, German, Japanese, Marathi, Telegu, Punjabi, Tamil, Turkish, Vietnamese, English, English
+		const match = regex.match(`
+			Hello World!
+			#圣诞节快乐
+			$क्रिसमसकीबधाई
+			$café
+			$bonneaprèsmidi
+			#Добрыйдень
+			$Straße
+			$こんにちは
+			$शुभदुपार
+			#శుభమద్యాహ్నం
+			$ਸਤਸ੍ਰੀਅਕਾਲ
+			$மதியவணக்கம்
+			#Tünaydın,
+			$tiếngchào
+			#test
+			$21e8`, 
+		'HASHTAG_REGEX');
+		console.log({ match }, match.length);
+		expect(!!match).toBe(true);
+	});
```

Comment:
This test would pass if only one of the hashtags in this matched. Can you break it out into multiple tests or check on how many of the lines matched

Reply:
Addressed in below commit.